mkdocs.yml

```diff
@@ -101,6 +101,7 @@ nav:
     - Apache NiFi: tools-and-applications/apache-nifi/apache-nifi.md
     - MLflow: tools-and-applications/mlflow/mlflow.md
     - Rclone: tools-and-applications/rclone/rclone.md
+    - Airflow: tools-and-applications/airflow/airflow.md
```

Comment:
Rename to Apache Airflow and move on first place (alphabetically sorted)

Reply:
Changed